bin/theme-setup.js

```diff
@@ -131,21 +133,157 @@ const promptThemeDataShort = ({devUrl, noConfirm}) => {
     }
 
     confirmed = summary([
+      { label: `${emoji.get('earth_africa')} Dev url`, variable: themeData.url },
+    ], noConfirm);
+  } while (confirmed !== 'y');
+
+  return themeData;
+};
+
+/**
+ * Prompts the user for all theme data (doesn't assume almost anything)
+ */
+const promptThemeData = ({ themeName, devUrl, noConfirm }) => {
+  let confirmed = 'n';
+  const themeData = {};
+
+  // -----------------------------
+  //  Prompt for project info
+  // -----------------------------
+
+  do {
+    if (!themeName) {
+      themeData.name = promptFor({
+        label: `${emoji.get('green_book')} Please enter your theme name (shown in WordPress admin):`,
+        prompt: 'Theme name: ',
+        error: 'Theme name field is required and cannot be empty.',
+        minLength: 2,
+      }).trim();
+    } else {
+      themeData.name = themeName;
+    }
+
+    // Build package name from theme name
+    themeData.package = themeData.name.toLowerCase().split(' ').join('_');
+
+    // Build prefix from theme name using one of 2 methods.
+    // 1. If theme name has 2 or mor more words, use first letters of each word
+    themeData.prefix = '';
+    const themeNameWords = themeData.name.split(' ');
+    if (themeNameWords && themeNameWords.length >= 2) {
+      for (const word of themeNameWords) {
+        themeData.prefix += word.charAt(0).toUpperCase();
+      }
+    }
+
+    // 2. If theme has only 1 word, use the first 3 letters of theme name
+    if (themeData.prefix.length < 2 && themeData.name.length > 2) {
+      themeData.prefix = (`${themeData.name.charAt(0)}${themeData.name.charAt(1)}${themeData.name.charAt(2)}`).toUpperCase();
+    }
+
+    themeData.env = `${themeData.prefix}_ENV`;
+    themeData.assetManifest = `${themeData.prefix}_ASSETS_MANIFEST`;
+
+    // Namespace
+    themeData.namespace = capCase(themeData.package);
+  
+    // Dev url
+    if (!devUrl) {
```

Comment:
Same as the first comment with `if/else`. A bit weird that linter didn't warn about this, it usually throws warnings for this 🤷‍♂

Reply:
Yup makes sense. Linter doesn't say anything about it when working on the repo (no `node_modules`) and my test project isn't setup to use it (should probably fix that). 🤷‍♂